projects/github-actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -162,7 +162,7 @@ async function getLabelsToAdd( octokit, owner, repo, number, isDraft ) {
 			keywords.add( '[Tools] Development CLI' );
 		}
 
-		const docs = file.match( /^docs\// );
+		const docs = file.match( /^docs\/|\.md$/ );
```

Comment:
Probably don't want to match CHANGELOG.md though.

And maybe also not the fixtures in `projects/packages/changelogger/tests/php/tests/lib/fixtures/`, although it's not terribly likely we'll add or change any of those anymore.

We could do the changelog part easily enough like this
```suggestion
		const docs = file.match( /^docs\/|(?<!\/CHANGELOG)\.md$/ );
```
although it'd probably be clearer to just do
```suggestion
		const docs = file.match( /^docs\/|\.md$/ ) && ! file.match( /\/CHANGELOG\.md$/ );
```
The second way would also make it easier to add in `|\/tests\/` or `|\/fixtures\/` or whatever seems appropriate if we want to handle that case too.